store/cache/cache.go

```diff
@@ -0,0 +1,38 @@
+package cache
+
+import (
+	"context"
+	"errors"
+
+	logging "github.com/ipfs/go-log/v2"
+	"go.opentelemetry.io/otel"
+
+	eds "github.com/celestiaorg/celestia-node/share/new_eds"
+)
+
+var (
+	log   = logging.Logger("store/cache")
+	meter = otel.Meter("store_cache")
```

Comment:
```suggestion
	log   = logging.Logger("eds:cache")
	meter = otel.Meter("eds:cache")
```

Reply:
assuming store moves as `eds` subpkg(it should)

Reply:
sync: decided to defer package structure discussion for later which will involve logger name